node/_events.js

```diff
@@ -37,7 +37,7 @@ import {
   validateAbortSignal,
   validateBoolean,
   validateFunction,
-} from "./_validators.ts";
+} from "./internal/validators.js";
```

Comment:
All this renames might seem unnecessary, but this actually solves two problems:

1) _validators is actually a shim of internal/validators however it was left in that location in the early days of std/node, now it will be easier to find the required functions when porting modules
2) This solves a circular dependency created between _validators and internal/validators by moving the content of _validators

Reply:
Thanks, it was bothering me too; good change.